docs/source/index.rst

```diff
@@ -23,14 +23,13 @@ Driver Configuration
 ====================
 
 The DNP3 driver configuration file follows `Driver Configuration File <https://volttron.readthedocs.io/en/main/agent-framework/driver-framework/platform-driver/platform-driver.html?highlight=driver_config#driver-configuration-file>`_ convention.
-Within the DNP3 driver configuration file, the "driver_config" argument is a key-value dictionary used to initialized
-a DNP3 outstation object class:
-    * master_ip: master station (driver) ip address
-    * outstation_ip: outstation (dnp3-agent) ip address
-    * master_id: master station ID
-    * outstation_id: outstation ID
-    * port: port number
-(For more details about DNP3 outstation object clas, please see `MyOutStationNew <https://github.com/VOLTTRON/dnp3-python/blob/main/src/dnp3_python/dnp3station/outstation_new.py#L35>`_.)
+Within the DNP3 driver configuration file, the "driver_config" argument is a key-value dictionary used to establish
+communication with a DNP3 outstation:
+    - **master_ip** - master_ip: master station (driver host) ip address
```

Comment:
I'm getting an indentation error here for some reason. I think there needs to be a blank line before the bullet list>